src/migrations/201602231400_initial.js

```diff
@@ -17,7 +17,7 @@ function createAccountsTable (knex) {
 function createFulfillmentsTable (knex) {
   return knex.schema.createTableIfNotExists('fulfillments', (table) => {
     table.increments()
-    table.uuid('transfer_id').unique().index()
+    table.uuid('transfer_id').unique()
```

Comment:
That's an index on a different table, so it won't index this column. If we don't index this it will probably cause performance problems, so I think we have to figure it out.